job_definitions/rotate_ft_account_passwords.yml

```diff
@@ -81,18 +81,4 @@
               scripts/rotate-api-tokens.sh sync-ft-account-passwords $STAGE
           ''')
         }
-        stage("Re-enable functional/smoke/smoulder tests") {
```

Comment:
Is `make jenkins TAGS=jobs JOBS=smoke_smoulder_tests` the thing that makes this redundant?

Reply:
Never mind - just seen the linked ticket in the commit description. Makes sense now.